src/odemis/driver/orsay.py

```diff
@@ -524,7 +543,12 @@ def moveAbs(self, pos):
         Move the axis of this actuator to pos.
         """
         self._checkMoveAbs(pos)
-        return self._executor.submit(self._changeVacuum, goal=pos["vacuum"])
+
+        if "vacuum" in pos:
+            vac_status = util.index_closest(pos["vacuum"], VACUUM_STATUS_TO_PRESSURE)
+            return self._executor.submit(self._changeVacuum, goal=vac_status)
+        else:
+            return InstantaneousFuture()
```

Comment:
In the logging message on line 533 'choices[goal]' should be 'choices[VACUUM_STATUS_TO_PRESSURE[goal]]'
I can't add this comment at that line, so I placed it here.